src/components/CurrentAddress/index.js

```diff
@@ -143,6 +143,10 @@ class CurrentAddress extends PureComponent<Props, { copyFeedback: boolean }> {
     copyFeedback: false,
   }
 
+  componentWillUnmount() {
+    this._isUnmounted = true
+  }
```

Comment:
can we do a clearTimeout instead (and putting the inner closure in a method?)

Reply:
yep! better. thx